src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -228,13 +225,7 @@ private async Task UpdateJobRecordAsync(CancellationToken cancellationToken)
             List<Tuple<string, string>> queryParametersList = new List<Tuple<string, string>>(sharedQueryParametersList);
             if (progress.ContinuationToken != null)
             {
-                var continuationToken = progress.ContinuationToken;
-                if (!Base64FormatRegex.IsMatch(continuationToken))
-                {
-                    continuationToken = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(continuationToken));
-                }
-
-                queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.ContinuationToken, continuationToken));
+                queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.ContinuationToken, progress.ContinuationToken));
```

Comment:
>progress.ContinuationToken [](start = 97, length = 26)

Can I ask why?
Was decoding added accidently?

Reply:
It was added as part of #591. Now that the change has been live for a few weeks there shouldn't be any old continuation tokens still floating around. This is just the first change in this area since then, so I'm cleaning it up.

Reply:
Any chance you can add description of that change and reasoning for that in PR description?
In case if someone have to blame file, and check changes, going through comments in PR is a bit too much for ask...